.github/workflows/ci.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
       matrix:
         os : [ 
           ubuntu-latest
-          # org-rancher-arm64-containers
+          # org-rancher--arm64-containers
```

Comment:
OK, I have made those changes. Let's see. 🤞🏽

Reply:
I made one mistake though. Forgot the extra `-` in the `build` step. Have fixed that now. Btw, how do you figure out that the queue for runners is long? I mean, is it visible on some status page or something like that?

Reply:
I really wish there was a single page to check the queue - mostly I was going off intuition from seeing other `rancher/*` repos I'm working in having long queue times too. On each action that's queued it will tell you how long it's been waiting.

Sometimes if you're iterating rapidly and waiting on multiple layers of tests in the queue it's worth cancelling all but the newest test. That way you don't bother running CI for old commits if there is a newer one ahead of it that will be tested too. To do that normally I just navigate to the Action tab of the repo, then filter for "queued" status. And then carefully cancel only ones related to the PR you're working on. 😄